packages/typescript/chat-core/test/threads/thread-client.test.ts

```
@@ -0,0 +1,430 @@
+import { describe, it, expect, beforeEach, vi } from 'vitest';
+import { ThreadUtilityClient, createThreadUtilityClient } from '../../src/thread-client';
+import {
+  createMockChatCoreAdapter,
+  createMockUserContext,
+  createMockGetUserContext,
+  createMockRequest,
+  createMockThread,
+  createMockThreadsResponse,
+  testScenarios,
+  errorScenarios,
+  expectToHaveBeenCalledWithUserContext,
```

ESLint

[error] 'expectToHaveBeenCalledWithUserContext' is defined but never used.

(@typescript-eslint/no-unused-vars)